docs/en/sql-reference/functions/geo/polygon.md

```diff
@@ -4,6 +4,34 @@ sidebar_label: Polygons
 title: "Functions for Working with Polygons"
 ---
 
+## WKT
+
+Returns a WKT (Well Known Text) geometric object from tuples which represent point coordinates.
```

Comment:
This documentation is incomplete - we need to explain behavior for various geometric types and various shapes of arrays.

Reply:
@alexey-milovidov thanks, I have pushed the update. How does it look now?